src/GeneralStateTestsFiller/stBadOpcode/eip2315NotRemovedFiller.json

```diff
@@ -28,7 +27,7 @@
       "1000000000000000000000000000000000000000" : {
         "balance" : "1000000000000000000",
         "//code" : "This should jump into a subroutine, back out and stop.",
-        "code" : "(asm 0x04 JUMPSUB STOP BEGINSUB RETURNSUB)",
+        "code" : "(asm 0x01 0x01 sstore 0x04 JUMPSUB STOP BEGINSUB RETURNSUB)",
```

Comment:
if BEGINSUB and RETURNSUB are reverted then lll support for this should also be reverted. 

need to unmerge this PR https://github.com/winsvega/solidity/pull/2
into PR state again. (git reset --soft HEAD~1)